lib/segment/Cargo.toml

```diff
@@ -19,7 +19,7 @@ openblas-src = { version = "0.10", default-features = false, features = ["cblas"
 
 parking_lot = "0.11"
 itertools = "0.10"
-rocksdb = "0.15.0"
+rocksdb = { version = "0.15.0", default-features = false, features = [ "snappy" ] }
```

Comment:
In order to prevent the following errors:
```
          /usr/bin/ld: /home/grekon/projects/rust/qdrant/target/debug/deps/libzstd_sys-ce6ce23fa380458c.rlib(fse_decompress.o): in function `FSE_decompress':
          /home/grekon/.cargo/registry/src/github.com-1ecc6299db9ec823/zstd-sys-1.5.0+zstd.1.4.9/zstd/lib/common/fse_decompress.c:385: multiple definition of `FSE_decompress'; /home/grekon/projects/rust/qdrant/target/debug/deps/liblibrocksdb_sys-a4efe358b53b4892.rlib(fse_decompress.o):/home/grekon/.cargo/registry/src/github.com-1ecc6299db9ec823/librocksdb-sys-6.17.3/zstd/lib/common/fse_decompress.c:302: first defined here
```

This is because new actix brings ``zstd-sys`` crate and ``librocksdb-sys`` brings ``zstd`` source code (not a  ``zstd-sys`` crate) and hance it is compiled twice. But we do not use zstd compression in rocksdb as by default it uses snappy, so we do not need the other 3 compression features (including zstd) provided by default at the moment anyway

Not sure how it compiled after actix update, but after the changes in the source code only it started failing. Most likely it depends on a build order of dependencies and we could get lucky last time